draft-yasskin-http-origin-signed-responses.md

```diff
@@ -302,26 +302,23 @@ the headers as this serialization. Instead, this section defines a CBOR
 representation that can be embedded into other CBOR, canonically serialized
 ({{canonical-cbor}}), and then signed.
 
-The CBOR representation of an exchange `exchange`'s headers is the CBOR
-({{!RFC7049}}) array with the following content:
+The CBOR representation of a set of request and response metadata and headers is
+the CBOR ({{!RFC7049}}) array with the following content:
 
 1. The map mapping:
-   * The byte string ':method' to the byte string containing `exchange`'s
-     request's method.
-   * The byte string ':url' to the byte string containing `exchange`'s request's
```

Comment:
I was wondering if we'd need to fix the blow example, that includes ':url' in cbor representation.

Reply:
Good catch. I'll file a separate PR for this

Reply:
Filed #290